src/App.jsx

```diff
@@ -1,3 +1,15 @@
-function App() {}
+import { useState } from "react";
+import ArticlesList from "./components.jsx/ArticlesList";
+import NavBar from "./components.jsx/NavBar";
+
+function App() {
+  const [topic, setTopic] = useState("");
```

Comment:
Try not to add any code that isn't required for the ticket that you are working on - this code is not relevant to this ticket